src/types/measure.d.ts

```diff
@@ -0,0 +1,18 @@
+export interface Measure {
+  id: {
+    timestamp: number;
+    date: string;
+  };
+  measureHumanReadableId: string;
+  measureSetId: string;
+  version: number;
+  revisionNumber: number;
+  state: string;
+  name: string;
```

Comment:
`name` was recently changed to `measureName`.

Reply:
I don't think we require this, since we have a new "models" directory to handle typescript types.

Reply:
I just removed it and updated the PR